pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -2382,7 +2382,6 @@ WorkflowNextSuggestion.StudyRedirect:
     - type: object
       required:
         - studyName
-        - studyGuid
```

Comment:
Another way to do this is keep this here in `required` but mark it as `nullable: true` in `properties`. And then you can update `description` to mention that it might not be available if it's not a pepper study.